bgpd/bgp_vty.c

```diff
@@ -1404,13 +1463,20 @@ DEFUN (no_bgp_cluster_id,
        "Route-Reflector Cluster-id in IP address format\n"
        "Route-Reflector Cluster-id as 32 bit quantity\n")
 {
-	VTY_DECLVAR_CONTEXT(bgp, bgp);
-	bgp_cluster_id_unset(bgp);
-	bgp_clear_star_soft_out(vty, bgp->name);
+	int idx_ipv4 = 3;
+	const char *router_id_str = NULL;
 
-	return CMD_SUCCESS;
+	if (argc == 4)
+		router_id_str = argv[idx_ipv4]->arg;
+
+	nb_cli_enqueue_change(vty,
+			      "./route-reflector/route-reflector-cluster-id",
+			      NB_OP_DESTROY, router_id_str);
+
+	return nb_cli_apply_changes(vty, NULL);
 }
 
+
 DEFUN (bgp_confederation_identifier,
```

Comment:
All converted commands need to be renamed to `DEFUN_YANG` or any of its varations (e.g. `DEFPY_YANG`, `DEFUN_YANG_NOSH`, etc). This is necessary for the NB back-off timer hack to work properly (#6727).